Minecraft/Player.ts

```diff
@@ -0,0 +1,18 @@
+import { read } from 'jimp';
+
+export class Player {
+    public name: string;
+    public uuid: string;
+    private image: any;
+
+    constructor(name: string, uuid: string) {
+        this.name = name;
+        this.uuid = uuid
+    }
+
+    async downloadImage() {
+        if(this.image) return this.image;
+        this.image = await read(`https://mc-heads.net/avatar/${this.uuid || this.name}`);
```

Comment:
Might be a better idea to use the skin data url sent in the player list packet (Mojang CDN) and extract the head ourself instead of spamming this api.

Reply:
I was a bit confused by the documentation I found on wiki.vg for the mojang api but I realize now that I’m supposed to decode the base64 response. I will change it to the mojang api once I get on my pc

Reply:
I have gone ahead and decided to use this API. Using the official Mojang API is a good idea however after playing around with it a bit it would require us to make an http server, and when using jimp the quality and proportions were incorrect and there appeared to be no solution.